flink-optimizer/src/main/java/org/apache/flink/optimizer/dataproperties/GlobalProperties.java

```diff
@@ -92,6 +95,10 @@ public void setRangePartitioned(Ordering ordering) {
 		this.partitioningFields = ordering.getInvolvedIndexes();
 	}
 	
+	public void setDataDistribution(DataDistribution distribution) {
```

Comment:
Can you remove the `setDataDistribution` method and add a parameter to `setRangePartitioned()` instead. Also add JavaDocs to `setRangePartitioned` to explain that the distribution is optional and might be null.